### ISSUE TITLE
get_next_simulation moved from DB to TAPAS code

### ISSUE THREAD

**Issue by schakalakka** [DLR-VF/TAPAS#47]
We moved the get_next_simulation function from the DB to TAPAS

Changes in the functionality. 
Before: All available servers were assigned to the first running simulation. (= one simulation in parallel)
After: A running simulation is only assigned to one server. (= several simulations in parallel)

Bug: Sometimes there seems to be a bug where one simulation is assigned to several servers. Probably because they ask for a new simulation (almost) at the same time. The method implemented in TAPAS is a hotfix for now. Needs to be improved. 

Nice to have improvement: Maybe have an assignment of simulations to servers (good for debugging too). 
Example: - Assign full berlin simulation to a powerfull server. 
- Assign a small 1% sample simulation to your local machine - for testing and debugging.
